packages/api/docs/future/api.yaml

```diff
@@ -0,0 +1,30 @@
+openapi: 3.0.0
+info:
+  title: Permanent Core API
+  version: 1.0.0
+  license:
+    name: AGPL-3.0
+    url: https://www.gnu.org/licenses/agpl-3.0.html
+servers:
+  - url: https://api.permanent.org/v2
+components:
+  securitySchemes:
+    bearerHttpAuthentication:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+paths:
+  /login:
+    $ref: "./paths/authentication.yaml#/login"
```

Comment:
I am *loving* the way you're breaking the spec into files instead of having a single gigantic swagger specification.

Very mindful, very demure.